### DISCUSSION TITLE
更新卡在53%

### DISCUSSION THREAD

**Discussion by Hao0912** [ywmoyue/biliuwp-lite#632] [Q&A]
### Bug 描述

从28版本更新到30.3，点更新后，一直卡在53%

### 操作步骤

```plain text
·
```

### 期望结果

·

### 截图

<img width="488" alt="Screenshot_2024-05-17_00-37-46" src="https://github.com/ywmoyue/biliuwp-lite/assets/149394777/6c9aec23-f800-4d85-b107-16a0527f134e">

### 应用来源

Github/其他Github镜像站

### 代码贡献

我更擅长发现问题

**Comment by ywmoyue** [comment]
引入了新的库，建议用脚本方式安装，否则依赖可能装不上
